sentry/src/main/java/io/sentry/clientreport/ClientReport.java

```diff
@@ -0,0 +1,122 @@
+package io.sentry.clientreport;
+
+import io.sentry.DateUtils;
+import io.sentry.ILogger;
+import io.sentry.JsonDeserializer;
+import io.sentry.JsonObjectReader;
+import io.sentry.JsonObjectWriter;
+import io.sentry.JsonSerializable;
+import io.sentry.JsonUnknown;
+import io.sentry.SentryLevel;
+import io.sentry.vendor.gson.stream.JsonToken;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class ClientReport implements JsonUnknown, JsonSerializable {
+
+  private final @NotNull Date timestamp;
+  private final @NotNull List<DiscardedEvent> discardedEvents;
+  private @Nullable Map<String, Object> unknown;
```

Comment:
I'm curious. What is this `unknown` map for?

Reply:
if new properties are added on Flutter, React Native or Sentry Native, this SDK will keep them and not lose during serialisation round trip

Reply:
Looking at the code i'd say it's used for extra JSON properties which we don't handle directly (yet).

Reply:
Ah okay thanks.